NEWS

```diff
@@ -38,9 +38,11 @@ Changed Functionality
   published vectors with holes. A reporter error is produced at runtime when
   serialization of vectors with holes is attempted.
 
-- Using libkrb5 for Kerberos support is now allowed on macOS. Due to very old age and
-  brokenness, the system version of the library is unsupported and will fail to
-  configure. Use the version from Homebrew or another newer installation.
+- Using libkrb5 for Kerberos support is now allowed on macOS. The default system library
+  previded by macOS is a very old version and may cause Zeek to crash. ``configure`` will
+  attempt to find the version installed by Homebrew first, but will report a warning and
+  disable Kerberos support if the system library is found. The ``--with-krb5`` option for
+  ``configure`` can be used To override using the version from Hombrew, if desired.
```

Comment:
Thanks for the rework!

There's a few typos and I wonder if it could be shortened:

> Kerberos support on macOS has been enabled. Due to incompatibilities, the system provided libkrb5 is ignored, however. Only versions from homebrew are supported and found/picked-up by default. Use ``--with-krb5`` for pointing at a custom librkb5 installation.